### DISCUSSION TITLE
Multiple projects reference for a test project

### DISCUSSION THREAD

**Discussion by joycezhaomeng** [stryker-mutator/stryker-net#2358] [Q&A] [answered]
My test project reference 4 projects in the same solution, but in configuration only can set one project, is there any way to work?
By the way, I have got CompileError when run Stryker, console output is `6     mutants got status CompileError. Reason: Mutant caused compile errors`, is there anyway to see the error detail?

**Comment by MischaVreeburg** [comment, marked as answer]
have a look at this project https://github.com/rajbos/Stryker.MultipleProjectRunner
You need to either manually create a config file or use powershell to create the file for you.
I opted to do the last.
automation:
- scan solution file for test projects 
- read the project references include paths. Make the absolute
- create pairs: one for each project reference found in the test project.
- create the config file and follow the instructions

**Reply by joycezhaomeng** [reply]
Thanks, I run the sperate test project one by one. And how about multi-reference projects for one test project? TestProjectA dependencies projectB and projectC? In the configuration, it only support one project, projectB or projecC both ok for the project config section` "project":"projectB"`?

**Reply by joycezhaomeng** [reply]
My test project reference projectA, and the projectA reference projectB and projectC. The mutation result is only for the projectA, and the score is 0.
Does it support to get the mutation result for the projectB and projectC?

**Reply by rouke-broersma** [reply]
No, we do not mutating of transitive project references. If you wish to mutate project b, add a project reference to b in the test project and target project b.

**Comment by rouke-broersma** [comment]
Compile errors are just informational, you don't need to do anything with this.

**Reply by joycezhaomeng** [reply]
thanks, I got the detail report. Could seed the compile errors in the report.
